facdb_build/sql/dcp_colp.sql

```diff
@@ -1,20 +1,21 @@
 --select w.status::text, count(*) 
---from (select geo::json->'status' as status from dcas_colp) w
+--from (select geo::json->'status' as status from dcp_colp) w
 --group by w.status::text;
-CREATE TABLE dcas_colp_tmp as
-	SELECT *
-	FROM dcas_colp
-	WHERE colp_type ~* 'maintenance|storage|Infrastructure|Office|residential|no use|private' 
-		 OR usedec ~* 'Office Leased By Educ|Emergency|Safety|Police|security|courthouse|ambulance station|OTHER SECURE DETENTION FACILITY'
-		 OR name ~* 'DETENTION FACILITY'
-		 OR (usedec ~* 'JAIL' AND (name LIKE '%COURT%' OR name LIKE '% CT %'));
-
-DROP TABLE dcas_colp;
-
-ALTER TABLE dcas_colp_tmp
-RENAME TO dcas_colp;
-
-ALTER TABLE dcas_colp
+CREATE TABLE dcp_colp_tmp as (
+	SELECT * 
+	FROM dcp_colp 
+	WHERE CONCAT("CATEGORY","EXPANDCAT") IN ('11', '12', '14', '16', '17', '28','29','38') 
```

Comment:
If you tested this and it works then ignore this comment.  If you didn't, I think you'll need to change this to `WHERE CONCAT(CATEGORY,EXPANDCAT)`